Docs/source/usage/parameters.rst

```diff
@@ -1125,6 +1125,10 @@ following the algorithm given by `Perez et al. (Phys. Plasmas 19, 083104, 2012)
     This is then used in the rest of the input deck;
     in this documentation we use ``<collision_name>`` as a placeholder.
 
+* ``<collision_name>.type`` (`string`)
+    Set to ``pairwisecoulomb`` for pairwise Coulomb collisions. This is the
```

Comment:
To avoid duplication, this should say something like this:
```suggestion
    The type of collsion. The types implemented are ``pairwisecoulomb`` for pairwise Coulomb collisions and 
``background_mcc`` for collisions between particles and a neutral background. If not specified, it defaults to ``pairwisecoulomb``.
```